main/opcodes/stack-operations.zkasm

```diff
@@ -7,6 +7,15 @@
  *  - stack input: none
  *  - stack output: [pushed_value]
  */
```

Comment:
This comment applies to `opPUSH1` and further, `opPUSH0` should be above it and with its own comment.

Reply:
Agree !! We will add specific comment for `PUSH0`